examples/kubernetes-dns/dns-matchname.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  name: "fqdn"
+specs:
+  - endpointSelector:
+      matchLabels:
+        org: empire
+        class: mediabot
+    egress:
+    - toFQDNs:
+        - matchName: "api.twitter.com"  
+  - endpointSelector:
```

Comment:
This is duplicated, you can have one endpointSelector. egress is a list of rules so you don't have to use `specs:` either and can keep it simple in a single rule which allows two egress segments.

Reply:
ok.